Dockerfile-arm64

```diff
@@ -0,0 +1,67 @@
+FROM multiarch/ubuntu-core:arm64-bionic
```

Comment:
If the difference between `Dockerfile` and `Dockerfile-arm64` is only the base image, we can integrate them by supplying the base image name using `--build-arg`.